src/components/SummaryPanel/ResponseTimeChart.tsx

```diff
@@ -48,14 +53,20 @@ export default class ResponseTimeChart extends React.Component<ResponseTimeChart
             <div>
               <strong>{this.props.label}:</strong>
             </div>
-            <AreaChart
-              size={{ height: 80 }}
-              color={{ pattern: [PfColors.Black, PfColors.Green400, PfColors.Blue, PfColors.Orange400] }}
-              legend={{ show: true }}
-              grid={{ y: { show: false } }}
-              axis={axis}
-              data={chartData}
-            />
+            {(this.thereIsTrafficData() && (
```

Comment:
Minor: I think the ternary conditional syntax (i.e. ? conditional) would be more natural here.

Reply:
Yes, ternary is better. Changed.